src/Microsoft.Azure.WebJobs.ServiceBus/ServiceBusAccount.cs

```diff
@@ -12,28 +11,5 @@ internal class ServiceBusAccount
         public MessagingFactory MessagingFactory { get; set; }
 
         public NamespaceManager NamespaceManager { get; set; }
-
-        internal static string GetAccountOverrideOrNull(ParameterInfo parameter)
```

Comment:
Cross compiling TypeUtility allows me to reuse the single generalized method for this, removing duplication/special casing